syntax = "proto3";

package Signatures;

option go_package = "github.com/teslamotors/vehicle_command/pkg/protocol/protobuf/signatures";
option java_package = "com.tesla.generated.signatures";
option java_outer_classname = "RsaSignatureData";

message RsaSignatureData {
  fixed32  epoch = 1;
  bytes   tag   = 2;
}